docs/user/tutorial/40-people.rst

```diff
@@ -49,7 +49,11 @@ To create a new Person:
         Description
             Freda oversees all technical activities in our workshop.
 
-    .. todo:: screenshot of above
+.. image:: /user/tutorial/images/create_new_person.png
```

Comment:
Is it possible to make the image less high, so we can make it wider (and the text easier to see) without taking up too much space?